### PR TITLE
Misc test fixes

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: error_reporting/composer.json [hunk 1]
```diff
@@ -7,5 +7,10 @@
     "require-dev": {
         "phpunit/phpunit": "~4.8",
         "google/cloud-tools": "^0.6.9"
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Google\\Cloud\\Samples\\Monitoring\\": "test/lib"
```
Comment (tmatsuo) [inline error_reporting/composer.json]: The directory is `error_reporting` while the namespace is `Monitoring`?

File: error_reporting/test/error_reportingTest.php [hunk 2]
```diff
@@ -17,19 +17,13 @@
 
 namespace Google\Cloud\Samples\Monitoring;
```
Comment (tmatsuo) [inline error_reporting/test/error_reportingTest.php]: Should we match the namespace to the directory?

File: error_reporting/test/quickstartTest.php [hunk 3]
```diff
@@ -15,17 +15,13 @@
  * limitations under the License.
  */
 
-use Google\Auth\ApplicationDefaultCredentials;
-use GuzzleHttp\Client;
-use GuzzleHttp\HandlerStack;
-use Google\Cloud\TestUtils\EventuallyConsistentTestTrait;
+namespace Google\Cloud\Samples\Monitoring;
```
Comment (tmatsuo) [inline error_reporting/test/quickstartTest.php]: ditto

File: error_reporting/composer.json (hunk 1 above)
Comment (bshaffer) [reply error_reporting/composer.json]: Good catch! These should all be fixed now.

**Comment by tmatsuo** [comment]
The code looks good.

Do you know why one of the tests in logging was skipped?
https://travis-ci.org/GoogleCloudPlatform/php-docs-samples/jobs/337274790#L4223

**Comment by bshaffer** [comment]
@tmatsuo amazing!! Yes, I renamed a test from `testWrite` to `testWriteAndList` and forgot to update the next test which had `@depends testWrite`. This is now fixed.

**Pull request merged**